Source/Laser/LaserProfilesImpl/LaserProfileGaussian.cpp

```diff
@@ -0,0 +1,101 @@
+#include <LaserProfiles.H>
+
+#include <WarpX_Complex.H>
+#include <WarpXConst.H>
+
+using namespace amrex;
+using namespace WarpXLaserProfiles;
+
+void
+GaussianLaserProfile::init (
+    const amrex::ParmParse& ppl,
+    const amrex::ParmParse& ppc,
```

Comment:
this will cause an unused parameter warning in the compilation file, just write
```suggestion
    const amrex::ParmParse& /* ppc */,
```